### ISSUE TITLE
Error 400 Bad Request when signing up for notifications

### ISSUE THREAD

**Issue by jhamilton09** [CachetHQ/Cachet#2043]
Before submitting your issue, please make sure that you've checked the checkboxes below.

- [ ] I am running the [latest release](https://github.com/CachetHQ/Cachet/releases/latest) version of Cachet.
- [ ] I am running at least PHP 5.5.9. *You can check this by running `php -v`.*
- [ ] I have ran `rm -rf bootstrap/cache/*`.

What version of Cachet? 2.2.2
What database driver? MySQL? Postgres? SQLite? MySQL
What version of PHP? 7.0.3

### Expected behaviour

*Please describe what you're expecting to see happen.*
Visit the status page. Click the envelope icon next to the component you wish to subscribe to.
Receive subscription successful message.

### Actual behaviour

*Please describe what you're actually seeing happen.*

Visit the status page. Click the envelope icon next to the component you wish to subscribe to.
You may receive Error 400 Bad Request.
Hit the browser's back button and click more subscription icons and you will repeatedly receive the Error 400 Bad Request. Looking in the dashboard the subscriptions are successful, whatever error is being generated is not being handled gracefully.

### Steps to reproduce

*If your issue requires any specific steps to reproduce, please outline them here.*

Nothing specific. Have tried multiple browsers, incognito mode, different email addresses. Error is received consistently.

**Comment by TheoBearman** [comment]
Can you dump your logs into this issue so we can see what's going on behind the scenes?

**Comment by GrahamCampbell** [comment]
This has been fixed in a newer release. Please upgrade to the latest version.

**Comment by jhamilton09** [comment]
Thanks. I reviewed the release notes and did not see it listed.
We updated our Cachet and I see the new subscription with per-component options after. That is much cleaner than all of the envelope icons. The weird thing was upon updating, existing subscribers were sent an email to confirm their subscription and those links resulted in the error 400. Creating a new subscription though, the process worked smoothly.
